src/lib/support/MathUtils.h

```diff
@@ -87,6 +87,26 @@ constexpr inline uint32_t RoundUp(uint32_t a, uint32_t b)
     return RoundDown(a + b - 1, b);
 }
 
+/**
+ * Number of bits in a byte.
+ */
+#ifndef NBBY
+#define NBBY             8
```

Comment:
I general I think we should avoid adding #defines if we can since they could collide with application names.  How about just making this a constexpr uint32_t inside BitsToByteLength()?

Reply:
done